containers/server-systemd-services/uyuni-server.service

```diff
@@ -14,8 +14,8 @@ EnvironmentFile=-/etc/sysconfig/uyuni-server-systemd-services
 Restart=on-failure
 ExecStartPre=/bin/rm \
     -f %t/uyuni-server.pid %t/%n.ctr-id
+ExecStartPre=setup_podman_timezone.sh
 ExecStart=/usr/bin/podman run \
-    --tz=local \
```

Comment:
Why not use `--tz=${TZ}` and drop the `PassEnvironment=TZ`?  Wouldn't be easier and doing the same thing?

Reply:
mmm I don't think it's easier, since `local` cannot be used for TZ (just for `--tz`). So `--tz=${TZ}` would be the same of `PassEnvironment=TZ`